### PR TITLE
Spring Boot configuration file usage

### PR DESCRIPTION
**Pull request by jeanbisutti**

### CODE REVIEW

File: content/en/docs/languages/java/automatic/spring-boot.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ following cases:
 [Spring Boot starter]:
   https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.build-systems.starters
 
+You can use the Spring Boot configuration files (`application.properties`,
+`application.yml`) to configure the OpenTelemetry Spring Boot starter. You can't
+do this with the OpenTelemetry Java agent.
+
```
Comment (trask) [inline content/en/docs/languages/java/automatic/spring-boot.md:29]: just checking, did you want to add this to the list above?

Comment (jeanbisutti) [reply content/en/docs/languages/java/automatic/spring-boot.md:29]: for me, this is a weaker reason for using the OpenTelemetry starter than those mentioned above

Comment (trask) [reply content/en/docs/languages/java/automatic/spring-boot.md:29]: thanks, I think I'd suggest adding it to the end of the list, even if it's less important

* You can use the Spring Boot configuration files (`application.properties`,
`application.yml`) to configure the OpenTelemetry Spring Boot starter which doesn't work with the OpenTelemetry Java agent

**Review by zeitlinger — APPROVED** [review]
LGTM :+1:

**Comment by jeanbisutti** [comment]
@open-telemetry/docs-approvers PTAL

**Review by svrnm — APPROVED** [review]

**Review by trask — APPROVED** [review]

**Pull request merged**